model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -214,11 +214,13 @@ message ExecutableStagePayload {
   // Input PCollection id.
   string input = 2;
 
+  repeated string side_inputs = 3;
```

Comment:
Please add a comment indicating that these are global ids (as opposed to local names).

Reply:
Added comments like `input` and `outputs`